contracts/InterchainTokenService.sol

```diff
@@ -626,32 +621,59 @@ contract InterchainTokenService is
      * @param sourceAddress The address of the remote ITS where the transaction originates from.
      * @param payload The encoded data payload for the transaction.
      */
-    function execute(bytes32 commandId, string calldata sourceChain, string calldata sourceAddress, bytes calldata payload) public {
+    function execute(
+        bytes32 commandId,
+        string calldata sourceChain,
+        string calldata sourceAddress,
+        bytes calldata payload
+    ) external onlyRemoteService(sourceChain, sourceAddress) whenNotPaused {
```

Comment:
Hey I think we don't need that one as it's already modified with `onlyRemoteService` in `_execute`. But does it make sense to add `onlyRemoteService` to the `expressExecute`? So random contracts can't send express TXs?
cc @milapsheth

Reply:
`contractCallValue` checks it so our relayer shouldn't express (and it whitelists source address as well), so don't need to pay for the extra gas